server/commands/src/arguments.rs

```diff
@@ -42,6 +43,45 @@ impl ArgumentKind<CommandCtx> for EntitySelector {
     }
 }
 
+impl EntitySelector {
+    /// Parses the returned entities for use in reporting success messages
+    /// Either the name of the entity for one entity, or how many were affected for many entities.
+    pub fn entities_to_string(&self, ctx: &CommandCtx, add_player: bool) -> String {
```

Comment:
This could be moved to a `Display` impl so we can call `to_string()` or `format("{}", entities)` it.

Reply:
How can I implement Display/fmt and still have it pass the two parameters?
The "add_player" is probably not needed (makes the grammar the same as vanilla) but it needs the command context to work

Reply:
You're right, good point.